.github/workflows/checks.yml

```
@@ -10,6 +10,9 @@ on:
       - main
       - 'release/**'
 
+env:
+  GO_VERSION: '1.24.5'
```

```suggestion
  GO_VERSION: '1.24.6'
```
Just to sync versions